News_App/src/app/authentication/authentication.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from '@auth0/auth0-angular';
+import { Profile } from '../interfaces/profile';
+
+@Component({
+  selector: 'app-authentication',
+  templateUrl: './authentication.component.html',
+  styleUrls: ['./authentication.component.scss'],
+})
+export class AuthenticationComponent implements OnInit {
+  constructor(public auth: AuthService) {}
```

Comment:
What is it auth? If you use authService interface you should name property the same like "authService"

Reply:
renamed